widget/playground/src/components/MultiSelect/MultiSelect.types.ts

```diff
@@ -1,10 +1,32 @@
-import type { MultiListPropTypes } from '../MultiList/MultiList.types';
+import type { Asset, Token } from 'rango-sdk';
 
-export type PropTypes = Omit<MultiListPropTypes, 'showCategory'> & {
+export interface CommonListProps {
+  type: 'Blockchains' | 'Bridges' | 'DEXs' | 'Wallets';
+  defaultSelectedItems: string[];
+  list: MapSupportedList[];
+  onChange: (items?: string[]) => void;
+}
+
+export interface TokensListProps {
+  type: 'Tokens';
+  list: (Token & { checked: boolean })[];
+  blockchains: string[];
+  onChange: (items?: Asset[]) => void;
+}
+export type MuliSelectPropTypes = (TokensListProps | CommonListProps) & {
   value?: string[];
+  label: string;
+  icon: React.ReactNode;
 };
 
 export interface MultiSelectChipProps {
   label: string;
   variant?: 'contained' | 'outlined';
 }
+
+export type MapSupportedList = {
+  title: string;
+  logo: string;
+  name: string;
+  networks?: string[];
```

Comment:
They are different. One is for categorizing networks, like wallets and blockchains with categories such as EVM, UTXO, or OTHER. The other is used to configure blockchains at the top of the supported tokens list. But I changed the names.

Reply:
Is `networks` prop same as `blockchains` above?